logging/cluster-logging-release-notes.adoc

```diff
@@ -1,34 +1,194 @@
 :_content-type: ASSEMBLY
 [id="cluster-logging-release-notes"]
```

Comment:
From our checklist: `Heading anchors use the correct syntax: [id="module-name_{context}"]`